src/tinybox.vue

```diff
@@ -202,6 +202,30 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    /*
+     * Center the thumbnails' scrollbar to the clicked image
+     */
+    index(index) {
+      var elements, el, thumbs;
+
+      if (!this.noThumbs) {
+        Vue.nextTick(function() {
```

Comment:
Codacy found an issue: ['Vue' is not defined.](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=5742748)